mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -121,12 +117,6 @@ LogicalResult PopulateParams::populateDerived(RockGemmWrapperInterface op,
     gemmSize.n += gemmExtraPad->n;
   }
 
-  if (isa<Conv2DBwdDataOp>(op) &&
-      !(gemmSize.m % 32 == 0 && gemmSize.n % 32 == 0 && gemmSize.k % 4 == 0)) {
-    LLVM_DEBUG(llvm::dbgs() << "Invalid gemm sizes for backward data.\n");
-    return failure();
-  }
```

Comment:
Again very strange we'd just disable this for no good reason. Thanks for chasing this down.